### PR TITLE
Eve95 & eve96: Hide TV Settings Icon & Implemented UI for URL and Color

### PR DESCRIPTION
**Pull request by alchucam**
#### What does this PR do?
Hide TV Settings Icon until mouse hovered.
Implement URL UI for BuildStatus and BuildAF
Implement Color UI for Banner

Minor Addition:
Added backend cron tasks for Jenkins to retrieve relevant data of all Jenkins project; hence, simplified Jenkins call and removed TeamBuildStatus.

#### Who is reviewing it?

@vinamartin @alchucam @mcalcote @haydenudelson

@Bdthomson @adimka @samuelechu @djblue @gordocanchola

#### How should this be tested?
`make go` and changes content in edit mode

#### Screenshots

image
image
image
image

#### Checklist:
- [x] Documentation Updated

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: client/components/README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Components
 
+## Banner
+
+Displays banner in the top screen. It contains Logo and Clocks.
```
Comment (vinamartin) [inline client/components/README.md]: ✏️ Add a note about the color being editable

File: coverage/clover.xml [hunk 2]
```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (haydenudelson) [inline coverage/clover.xml]: ✏️ Do we need everything from the coverage folder?

File: coverage/lcov-report/base.css [hunk 3]
```diff
@@ -0,0 +1,224 @@
+body, html {
```
Comment (haydenudelson) [inline coverage/lcov-report/base.css]: ✏️ Do we need this stuff from coverage?

File: client/components/Settings/ColorPicker.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { SketchPicker } from 'react-color'
+
+class ColorPicker extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            color: {
+                r: '0',
```
[the remark this answers is not in the document]
Comment (haydenudelson) [reply client/components/Settings/ColorPicker.js]: Shouldn't the default be whatever the currently selected color is?

[the remark this answers is not in the document]
Comment (alchucam) [reply client/components/Settings/ColorPicker.js]: it gives a default color to start with in the UI. It's just a starting point, and the color won't change until the user select a color so it shouldn't matter.

**Comment by alchucam** [comment]
> GSR Wallboard breaks now & latest changes in DEV branch not reflected- otherwise looks good to me! I really like the color picker.
> gsrerror

This has been taken care of in https://github.com/connexta/eve/pull/99

**Pull request merged**